modules/lvms-about-adding-devices-to-a-vg.adoc

```diff
@@ -12,6 +12,8 @@ You can specify the device paths in the `deviceSelector.paths` field, the `devic
 
 You can also add the path to the RAID arrays to integrate the RAID arrays with {lvms}. For more information, see "Integrating RAID arrays with {lvms}".
```

Comment:
To avoid redundancy, I think you can remove the "You can" from the three paragraphs in a row and just start them this way:
- "Specify the device paths in the..."
- "To integrate the RAID arrays with {lvms}, add the path to the..."
- "Add encrypted devices to the volume group by enabling disk encryption..." or "To add encrypted devices to the volume group, enable disk encryption on the..."

Reply:
Hi @sabrinajess 
Thanks for the review. I think it will be best have "You can" as those are not mandatory tasks. So keeping it as is :)